packages/dd-trace/src/profiling/profiler.js

```diff
@@ -47,9 +48,10 @@ class Profiler extends EventEmitter {
     }
 
     try {
+      const nearOOMCallback = (profileType, profile) => this._nearOOMExport(profileType, profile)
       for (const profiler of config.profilers) {
         // TODO: move this out of Profiler when restoring sourcemap support
-        profiler.start({ mapper })
+        profiler.start({ mapper, nearOOMCallback })
```

Comment:
Could also do something like:

```suggestion
        profiler.start({
          mapper,
          nearOOMCallback: this._nearOOMExport.bind(this)
        })
```

Then you don't need the line above making a `nearOOMCallback` closure.

Reply:
Neat !